### PR TITLE
Deployment/jkw deploy setup

### LINKED ISSUE #47: Deploy Deploy Deploy
### Feature Categories:
- React
- Other: TravisCI, Heroku

**Describe the feature/Bug your gonna work on**
Deploy the frontend!

### PR DESCRIPTION
**Pull request by jkwest-93**
# Som Timer Pull Request Form

## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
- Adds TravisCI functionality & deploys to Heroku
- Closes #47 
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other, Please explain: deployment

## Has This Been Tested?
- [ ] No
- [x] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
- Run `npm test` in the terminal to see passing tests
# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+- node
+notifications:
+  email: false
+deploy:
+  provider: heroku
+  api_key:
+    secure: UBOI/3MMwCfApguv41ykkxNU742VrlUsSjYIwmYCqcSyfkhrSXN10640KeGwx59CURPlQQ7ke95iU+9haq6SUzzMDoHsDcxZU4xlApFRhGWLwQEpOlMf0dSlhLfdp25HBOlugvOx9cyr0FFETUCLAjMLmVvivdh1+1vFgw4ePE9EVFA3Brt69SYtvTftPfji/2+XDsWxnXtU+dLUm0MjVwUlyupu43dm3ROVhTuaPBRHBoKSuoi0jabPOrVG08ztTwb6srOjpJ5mXgEb7/BhPveScRaTMLGkA28AQdUJWkeXO9FD4TPmvli42wzS4kcVg4u5ul/FWDqRBiVLeY+lnjdM95QtHsvbVgD9Uq1NbqXlLA1XVKPS2CEfgHMiJnFqy4CH1EnoYAYRWHOm/GhGiNGGLnBfBYUW0yJgPAG2LFSUKt3fOPNjnMrkT0f/xVB0mf0s97gbdc6sBtW3RnJb7xgpsEFqILnKc7m/OhdvpDIAVRsvM6WkO9GQ4UGrfudqN2hXzksozfmdge9ABCKusnAl3nMcsaWFoRv2jh3GgOVgk+ntGT+S0xalHMnTiW/y/mB96QjI8ddsswU9Swbkvpd+7OkwPGeJQKYG7D0bUzkRPYeC/TTNGxEcyw+K87M3qhmEl+PEzmzETS8E3dpiV0Md2WnFV7h2jAjWrqbADGo=
+  app: som-timer
```
Comment (Abdeboskey) [inline .travis.yml:10]: That's a hawt key 🔥

**Review by rwilliams659 — APPROVED** [review]
Great job fixing up those tests quick. So exciting to get this deployed with CI!!!

**Review by Abdeboskey — APPROVED** [review]
This is so exciting

File: .travis.yml (hunk 1 above)
Comment (sienna-kopf) [reply .travis.yml:10]: Sweet that you got this up and running so fast!1

**Review by sienna-kopf — APPROVED** [review]

**Pull request merged**